lib/rspec-puppet-facts-unsupported/on_unsupported_os.rb

```diff
@@ -265,7 +265,8 @@ def shuffle_and_limit(os_facts_hash)
     end
 
     def describe_os(facts)
-      "#{facts[:operatingsystem].downcase}-" \
+      os = facts[:operatingsystem] ? facts[:operatingsystem].downcase : nil
```

Comment:
I don't think that we should return nil when there is no `:operatingsystem` set, here. It's then converted to string and we get: `'nil-something'`. Maybe better is to set to something specific like `'unknown'`

Reply:
Good idea! :) Change it to `unknown`